ios/FluentUI/Card/CardView.swift

```diff
@@ -475,6 +483,14 @@ open class CardView: UIView {
         NSLayoutConstraint.activate(layoutConstraints)
     }
 
+    private func setLargeContentTitle() {
```

Comment:
"set" prefix is usually for property setter in apple. consider using "updateLargeContentTitle"? also should it be more generic to support icon as well? or is it by design to show only the title?

Reply:
I did not realize I could add the icon, updated with the icon now! the pr description has new screen shots of what it looks like.

Reply:
I decided to separate into two helper methods so we only update the icon or title when the text or icon changes.